src/main/java/org/medicmobile/webapp/mobile/SmsSender.java

```diff
@@ -31,10 +41,17 @@ class SmsSender {
 
 	private final EmbeddedBrowserActivity parent;
 	private final SmsManager smsManager;
+	private Sms sms;
 
+	@SuppressWarnings("deprecation")
```

Comment:
Spotbugs lint still complaining about deprecated code when we're already handling with `Build.VERSION.SDK_INT >= XX` condition

Reply:
I first deleted the `build` folder, and run `make lint`

Reply:
I quick searched in spotbugs config but haven't found anything yet 🤔

Reply:
Ah it works but it's deprecated: 
https://spotbugs.readthedocs.io/en/latest/annotations.html?highlight=suppresswarnings#edu-umd-cs-findbugs-annotations-suppresswarnings-deprecated

Reply:
Funny, so I tried using `@SuppressFBWarnings` as they suggest but then the warning comes up again. With `@SuppressWarnings` the warning as gone. Then I noticed it's when it executes: `Task :compileUnbrandedDebugJavaWithJavac`

<img width="761" alt="Screen Shot 2022-04-20 at 2 15 38 pm" src="https://user-images.githubusercontent.com/66472237/164172043-ff5ffece-c156-4089-a8f7-9d6bb470a878.png">
<img width="1689" alt="Screen Shot 2022-04-20 at 2 16 04 pm" src="https://user-images.githubusercontent.com/66472237/164172130-8f187690-91f0-4ba4-bae3-7983ec1147c9.png">

Reply:
Okay, I think I see what I was missing. I was thinking that this was failing the build, but I think this is just the "normal" compiler warnings.  In that case, my vote would be to leave the warnings and not include the annotations. This is because we eventually want to update this code to not use the deprecated methods (after we raise our minSDK version). IMHO the only warnings we want to suppress are the ones where we know we do not want to change the way we are doing things (and so there is no reason to keep the warning around as a reminder...).  Ultimately, I am fine suppressing these if that makes more sense to you, though.